conda-store-server/conda_store_server/orm.py

```diff
@@ -197,14 +182,21 @@ class Build(Base):
     started_on = Column(DateTime, default=None)
     ended_on = Column(DateTime, default=None)
     deleted_on = Column(DateTime, default=None)
+
+    @staticmethod
+    def _get_build_key_version():
+        # Uses local import to make sure current version is initialized
+        from conda_store_server import BuildKey
+
+        return BuildKey.current_version()
```

Comment:
All `conda_store_server` imports are local in this file to avoid a cyclic import problem mentioned in a `BuildKey` class comment.